addons/web/static/src/views/fields/monetary/monetary_field.js

```diff
@@ -30,7 +30,7 @@ export class MonetaryField extends Component {
         useInputField({
             getValue: () => this.formattedValue,
             refName: "numpadDecimal",
-            parse: parseMonetary,
+            parse: this.parseMonetaryInput.bind(this),
```

Comment:
seems weird. can you explain why you need this? and if not, can't you just subclass the monetary field for your usecase?

Reply:
I extended the monetaryfield. See:
https://github.com/odoo/enterprise/pull/40068/files#diff-351714a7b99d580ceecdd5da8827a0897670c3ebdf409a676cbc690414b2c35aR3

Otherwise, I need a way to extend the parameter of `useInputField`